delocate/tools.py

```diff
@@ -1,132 +1,25 @@
 """ Tools for getting and setting install names """
 
-from subprocess import Popen, PIPE
-
-import os
-from os.path import join as pjoin, relpath, isdir, exists
-import zipfile
+from os.path import exists
 import re
-import stat
-import time
+
+from wheeltools.tools import (
+    back_tick,
+    ensure_writable,
+    find_package_dirs,
+    dir2zip,
+    zip2dir,
+    cmp_contents,
+    open_readable,
+    open_rw,
+    chmod_perms,
```

Comment:
@matthew-brett I kept these names in here as aliases for the functions that were moved out to the standalone wheeltools package. If you prefer, we can instead fix up all the relative `.tools` imports in delocate to instead import these from the `wheeltools.tools`.